packages/datadog-plugin-grpc/src/util.js

```diff
@@ -4,13 +4,16 @@ const pick = require('lodash.pick')
 const log = require('../../dd-trace/src/log')
 
 module.exports = {
-  addMethodTags (span, path, kind) {
```

Comment:
@rochdev we were using v2.4.0 in our repo but since some vuln was fixed in the later version we have to upgrade to v2.46.0.
Our repo used `addMethodTags` to add tags to the span. Since it was deprecated with this pr, and the tags are just returned  in `getMethodMetadata` is the caller expected to add the tags to the span? Moreover maintain the older format of tags for example:
```{
      'grpc.method.path': path,
      'grpc.method.kind': kind
    }
```

Reply:
This method was never exposed publicly, so if you were using it in your code I would say yes to adding the tags manually, which is the safe way to do it regardless of the existence of that method.

Reply:
Thank you! I understand. 
Is there a DD way to report the gRPC data? We would want to defer to that and not maintain a separate system.

Reply:
Are you using the auto-instrumentation? I guess I'm not sure I understand the use case since these should be added automatically in the first place.